# genome-references

## Variant Callers
[Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2)- Tumor/normal or tumor only variant calling with local realignment 
[Vardict](https://github.com/AstraZeneca-NGS/VarDict)- Tumor/normal mode or tumor only
[Strelka](https://github.com/Illumina/strelka)- Tumor/normal variant calling
[Neusomatic](https://github.com/bioinform/neusomatic)-Tumor/normal with standalone mode or ensemble calling mode
[Octopus](https://github.com/luntergroup/octopus)- Tumor/normal or tumor only


## RNA-Seq Variant Calling Pipelines
[CTAT-RNA-seek](https://github.com/NCIP/ctat-mutations/wiki/ctat_mutations_docker_singularity)  
[GATK Best Practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035531192-RNAseq-short-variant-discovery-SNPs-Indels-)  
[Deepvariant Germline RNA-seq ](https://github.com/google/deepvariant/blob/r1.5/docs/deepvariant-rnaseq-case-study.md)  

## scRNA-seq Variant calling
[cellsnp-lite](https://cellsnp-lite.readthedocs.io/en/latest/snp_list.html)- Uses population databases

