# genome-references

## Variant Callers
[Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2)- Tumor/normal or tumor only variant calling with local realignment  
[Vardict](https://github.com/AstraZeneca-NGS/VarDict)- Tumor/normal mode or tumor only  
[Strelka](https://github.com/Illumina/strelka)- Tumor/normal variant calling  
[Neusomatic](https://github.com/bioinform/neusomatic)-Tumor/normal with standalone mode or ensemble calling mode  
[Octopus](https://github.com/luntergroup/octopus)- Tumor/normal or tumor only  



## Contamination
[Conpair](https://github.com/nygenome/conpair)  


## Subclonality
[Clonevol](https://github.com/hdng/clonevol)- Visualization of results  
[Fastclone](https://github.com/GuanLab/FastClone_GuanLab)

## RNA-Seq Variant Calling Pipelines
[CTAT-RNA-seek](https://github.com/NCIP/ctat-mutations/wiki/ctat_mutations_docker_singularity)  
[GATK Best Practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035531192-RNAseq-short-variant-discovery-SNPs-Indels-)  
[Deepvariant Germline RNA-seq ](https://github.com/google/deepvariant/blob/r1.5/docs/deepvariant-rnaseq-case-study.md)  

## scRNA-seq Variant calling
[cellsnp-lite](https://cellsnp-lite.readthedocs.io/en/latest/snp_list.html)- Uses population databases  
[SCExecute](https://horvathlab.github.io/NGS/SCExecute/)  
[SComatic](https://github.com/cortes-ciriano-lab/SComatic)  
[scAllele](https://github.com/gxiaolab/scAllele)  
[cbSniffer](https://github.com/sridnona/cb_sniffer)  

### Others
[Tabular ML for VC](https://github.com/AbbVie-GRC-Methods-Dev/new_normal)  
[TOSCA](https://github.com/mdelcorvo/TOSCA)- Tumor Only Somatic Calling
